static/js/components/PrivacyTab.js

```diff
@@ -1,20 +1,18 @@
 import React from 'react';
 import Grid, { Cell } from 'react-mdl/lib/Grid';
 
+import ProfileProgressControls from './ProfileProgressControls';
 import ProfileFormFields from '../util/ProfileFormFields';
 
 class PrivacyTab extends ProfileFormFields {
```

Comment:
maybe not in scope for this PR, but a future refactor might be to simplify `ProfileFormFields` or get rid of it entirely. it's doing 2 things: (1) defining constant values for _all_ tabs, which it shouldn't be doing IMO, and (2) binding input elements to `this`. if we want a one-stop shop to make all form element types available to these tabs, then it would make some sense to keep `ProfileFormFields` around and to set it as some property in the tab class, eg:

```javascript
class PrivacyTab extends React.Component {
  constructor() {
    this.formFields = ProfileFormFields()
    //...
  }
  
  render() {
    return <div>{this.formFields.boundTextField()}</div>
  }
```

either way, we should really move the tab-specific values to their own tab classes. as for this specific tab class, it doesn't make semantic sense that `PrivacyTab` inherits from `ProfileFormFields`, and even less so now that all of the other tabs aren't subclassing it